### PR TITLE
First seed

### PR DESCRIPTION
**Pull request by Vivien-Ploix**
Creation first seed

### CODE REVIEW

**Comment by Vivien-Ploix** [comment]
[Création de la Seed Item et Restaurants](https://trello.com/c/F6ts3BpY/106-cr%C3%A9ation-de-la-seed-item-et-restaurants)

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,83 @@
...
+5.times do 
+  business = Business.create(
+    name: Faker::Restaurant.name,
+    address: Faker::Address.full_address,
+    phone: Faker::PhoneNumber.cell_phone,
+    photo: "http://placekitten.com/#{rand(150..200)}/#{rand(150..200)}",
+    owner_id: Owner.first.id,
+  )
+  businesses << business
+  puts businesses
```
Comment (mvoland) [inline db/seeds.rb:67]:
```suggestion
  puts business.name
```
Why the array, why puts the array?

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,83 @@
...
+HappyPrice.destroy_all
+Item.destroy_all
+Business.destroy_all
+Owner.destroy_all
+Customer.destroy_all
+Administrator.destroy_all
+
+
+
+businesses = Array.new
```
Comment (mvoland) [inline db/seeds.rb:21]:
```suggestion
```
Why this array?

**Review by mvoland — APPROVED** [review]
Ok for me, but I don't get why there is the "businesses" array

**Comment by Vivien-Ploix** [comment]
> Ok for me, but I don't get why there is the "businesses" array

I found it easier to get the business_id in the item seed, but it might not be the best option. As for the puts, they were only there to try and locate the error in the beginning, and I forgot to remove them, I'm going to change it

**Comment by mvoland** [comment]
Yes Business.sample should work (and categories.sample)
But I actually didn't see where businesses was used so I'm OK with this version.

**Pull request merged**